app/views/users/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<%= form_for @user, url:  "/users/#{@user.id}", method: :patch do |f| %>
+  <%= label_tag(:username) %>
```

Comment:
This should be written like this `<%= u.label :username %>`

Note:implement changes to all the label tags